projects/element-ng/help-button/si-help-button.component.scss

```diff
@@ -0,0 +1,25 @@
+@use '@siemens/element-theme/src/styles/variables';
+@use 'sass:map';
+
+:host {
+  display: inline-flex;
+  vertical-align: middle;
+  align-items: center;
+  justify-content: center;
+  block-size: 1rem;
+  inline-size: 1rem;
+  font-size: 1.331rem;
+  border-radius: 1rem;
```

Comment:
Considering those values are all affecting each other and I somewhat expect that we'll probably have other scales of help icons, WDYT if we factor this value out into a proper SCSS variable (e.g. `$si-hel-icon-size`)?

Reply:
I managed to drop most of them by using css transform instead. So if someone wants another size, he just has to set a different font-size. The rest can stay as is